docs/developer-guide/implement-a-udf.rst

```diff
@@ -0,0 +1,349 @@
+.. _implement-a-udf:
+
+Implement a User-defined Function (UDF)
+#######################################
+
+Prerequisites
+     - `Apache Maven <https://maven.apache.org/download.cgi>`__
+     - |cp| installed locally
+     - Internet connectivity for downloading Confluent POM files
+
+Create the KSQL Extensions Directory
+************************************
+
+When you create a custom user-defined function (UDF), you implement it in Java
+and deploy it as a JAR to the KSQL extensions directory. By default, this 
+directory doesn't exist, so you need to create it and assign it in the KSQL

```

Comment:
Can you raise a ticket against the tools team to have the `ext` directory created and have it set in the ksql-server.properties by default, thereby simplifying these instructions...?

Reply:
Apparently @hjafarpour knows about the packaging stuff.